extensions/extensions.json

```diff
@@ -110,6 +105,51 @@
             "website": "https://github.com/tillbaum",
             "image": "",
             "dependencies": []
+        },
+        {
+            "builtin": "False",
+            "type": "extension",
+            "rocket_mode_compatible": "True",
+            "name": "EF-Tools",
+            "description": "Set of custom tools.",
+            "author": "Erik Frits",
+            "author_profile": "https://www.linkedin.com/in/erik-frits/",
+            "url": "https://github.com/ErikFrits/EF-Tools.git",
+            "website": "https://www.erikfrits.com",
+            "image": "",
+            "dependencies": []
+        },
+        {
+            "builtin": "False",
+            "default_enabled": "True",
+            "type": "extension",
+            "rocket_mode_compatible": "True",
+            "name": "pyChilizer",
+            "description": "A collection of tools for Revit.",
+            "author": "Archilizer",
+            "author_profile": "https://archilizer.com",
+            "url": "https://github.com/dnenov/pyChilizer",
+            "website": "https://archilizer.com",
+            "image": "",
+            "dependencies": []
+        },
+        {
+            "builtin": "False",
+            "default_enabled": "True",
+            "type": "extension",
+            "rocket_mode_compatible": "True",
+            "name": "pySSG",
+            "description": "IronPython Scripts for Autodesk Revit.",
+            "author": "Kyle Bruxvoort",
+            "author_profile": "https://kbrux.com",
+            "url": "https://github.com/kbruxvoort/SSG.extension.git",
+            "website": "https://archilizer.com",
```

Comment:
I think this is the wrong website address @eirannejad

Reply:
Uh oh!

Reply:
I just now realized pySSG got added to the extensions list. Amazing!